cli/src/cli.rs

```diff
@@ -446,7 +446,7 @@ pub async fn mnemonic_command(output_file_name: Option<String>, output_stdout: O
 }
 
 pub async fn node_info_command(wallet: &Wallet) -> Result<(), Error> {
-    let node_info = serde_json::to_string_pretty(&wallet.client().get_info().await?)?;
+    let node_info = serde_json::to_string_pretty(&wallet.client().get_node_info().await?)?;
```

Comment:
wondering if should get rid of this method as we have the same in `wallet_cli/mod.rs`

Reply:
one requires you to have a wallet right

Reply:
they both require it

Reply:
Ah, then yeah I would dedup

Reply:
done